docs/en/ingest-management/elastic-agent/elastic-agent-providers.asciidoc

```diff
@@ -496,15 +488,15 @@ providers.kubernetes:
 ----
 
 `node`:: (Optional) Specify the node to scope {agent} to in case it
-cannot be accurately detected, as when running {agent} in host network
+cannot be accurately detected when running {agent} in host network
 mode.
 `cleanup_timeout`:: (Optional) Specify the time of inactivity before stopping the
-running configuration for a container, 60s by default.
-`sync_period`:: (Optional) Specify timeout for listing historical resources.
-`kube_config`:: (Optional) Use given config file as configuration for Kubernetes
-client. If kube_config is not set, KUBECONFIG environment variable will be
-checked and if not present it will fall back to InCluster.
-`scope`:: (Optional) Specify at what level autodiscover needs to be done at. `scope` can
+running configuration for a container. This is 60s by default.
+`sync_period`:: (Optional) Specify the timeout for listing historical resources.
+`kube_config`:: (Optional) Use the given config file as configuration for Kubernetes
+client. If kube_config is not set, the KUBECONFIG environment variable will be
+checked and will fall back to InCluster if not present.
+`scope`:: (Optional) Specify the level for autodiscover. `scope` can
```

Comment:
Does autodiscover need to be hyphenated?

Reply:
Good question. No need to hyphenate.